### PR TITLE
Eliminate `tJOY` and hide FM Banner

### LINKED ISSUE #3834: Update TJoy to Joy in Pioneer UI since we are in Carthage Release already
Hello Guys:

I just observed on Pioneer UI. For Carthage release, I think we should be using Joy instead of TJoy.
Please see picture on link below...

https://discord.com/channels/811216481340751934/1035124220012810280/1041979345448730665

Thanks.

### PR DESCRIPTION
**Pull request by traumschule**
Fixes #3834

### CODE REVIEW

File: packages/ui/src/app/App.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { Redirect, Route, Switch } from 'react-router-dom'
 
 import '@/services/i18n'
 
-import { FMBanner } from '@/app/components/FMBanner'
+//import { FMBanner } from '@/app/components/FMBanner'
```
Comment (freakstatic) [inline packages/ui/src/app/App.tsx:6]: Same

**Review by freakstatic — CHANGES_REQUESTED** [review]

**Review by freakstatic — COMMENTED** [review]
Shouldn't `FMBanner.tsx` be deleted?

**Comment by traumschule** [comment]
ok boss :)

**Review by freakstatic — APPROVED** [review]

**Comment by thesan** [comment]
@traumschule, @freakstatic sorry we completely missed this PR and the issue it was open to for. There is a different set of issues opened when Mainnet was planned and a another PR addressing them: #3859 which is already merged.

**Pull request closed without merging**